src/main/services/team/provisioning/__tests__/TeamProvisioningDiagnosticsHelpers.test.ts

```
@@ -0,0 +1,59 @@
+import type { TeamProvisioningSupportDiagnostic } from '@shared/types';
+import { describe, expect, it } from 'vitest';
+
+import {
+  buildProvisioningTraceDetail,
+  pushUniqueSupportDiagnostics,
+} from '../TeamProvisioningDiagnosticsHelpers';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)